lib/sidekiq/api.rb

```diff
@@ -5,7 +5,9 @@
 require "zlib"
 require "base64"
 
+# @api public
 module Sidekiq
+  # @api private
```

Comment:
This is private till we have more documentation on stats. Right now, we are a bit slim